pulp_rpm/tests/unit/test_modulemd.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+
+from pulp_rpm.app.modulemd import parse_modular
+
+
+def test_parse_modular_preserves_literal_unquoted_values_3285():
+    """Unquoted yaml numbers with trailing/leading zeros are preserved"""
+
+    file_path = str(Path("pulp_rpm/tests/unit/modulemd.yaml").absolute())
+    all, default, obsoletes = parse_modular(file_path)
```

Comment:
You should use the `tmp_path` fixture from pytest and then change into that directory before calling parse_modular()

```
def test_parse_modular_preserves_literal_unquoted_values_3285(tmp_path):
    """Unquoted yaml numbers with trailing/leading zeros are preserved"""

    os.chdir(tmp_path)
    file_path = str(Path("pulp_rpm/tests/unit/modulemd.yaml").absolute())
    all, default, obsoletes = parse_modular(file_path)
```

https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html